shoop/simple_pricing/models.py

```diff
@@ -18,7 +18,6 @@ class SimpleProductPrice(models.Model):
     shop = models.ForeignKey("shoop.Shop", blank=True, null=True, db_index=True)
     group = models.ForeignKey("shoop.ContactGroup", blank=True, null=True, db_index=True)
```

Comment:
Probably a good idea. (The migration will naturally need to get rid of the currently-null-or-blank SPPs before it edits the schema.)